src/components/InfoPanel.component.js

```diff
@@ -22,6 +22,11 @@ class InfoPanel extends Component {
   }
 
   render() {
+    const IMAGE_URL_1 =
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Travis_Scott_April_2017.jpg/1200px-Travis_Scott_April_2017.jpg'
+    const IMAGE_URL_2 =
+      'https://media.pitchfork.com/photos/592993405e6ef9596931ee5e/1:1/w_300/ebab43f0.jpg'
+    const IMAGE_URL_3 = 'https://assets.rbl.ms/8276008/980x.jpg'
```

Comment:
Since a lot of the pois in the db don't have real urls yet, using these temp images is okay for now. But since we will eventually have to use the urls from the pois (can do in another pr), it would be better to get as close to that implementation as possible. 

The actual poi media content will look something like this: 

```
[
  {
    "caption": "some text",
    "content_url": "http://something.com/image.jpg",
    "id": 2,
    "poi_link": 14
  },
  {
    "caption": "some text",
    "content_url": "http://something.com/image.jpg",
    "id": 2,
    "poi_link": 14
  }
]
```

You should adapt this component to work given this structure, so it'll fit right in when we use the real poi data instead of the dummy data